samza-core/src/test/java/org/apache/samza/task/TestStreamOperatorTask.java

```diff
@@ -19,26 +19,30 @@
 
 package org.apache.samza.task;
 
+import java.lang.reflect.Field;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import org.apache.samza.SamzaException;
 import org.apache.samza.context.Context;
 import org.apache.samza.context.JobContext;
 import org.apache.samza.operators.OperatorSpecGraph;
 import org.apache.samza.operators.impl.OperatorImplGraph;
 import org.apache.samza.system.IncomingMessageEnvelope;
+import org.apache.samza.system.SystemStream;
 import org.apache.samza.util.Clock;
 import org.junit.Test;
+import org.mockito.ArgumentCaptor;
 
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
There is mix and match of static star imports in the code. That said, we have settled on no star imports. Would you mind adding explicit imports instead? Maybe, clean the mockito ones in the file too :-)

Reply:
Changed it. I just assumed that it was okay as there were no checkstyle checks on star imports during build.